src/components/views/dialogs/security/SetupEncryptionDialog.tsx

```diff
@@ -10,55 +10,19 @@ import React from "react";
 
 import SetupEncryptionBody from "../../../structures/auth/SetupEncryptionBody";
 import BaseDialog from "../BaseDialog";
-import { _t } from "../../../../languageHandler";
-import { SetupEncryptionStore, Phase } from "../../../../stores/SetupEncryptionStore";
-
-function iconFromPhase(phase?: Phase): string {
-    if (phase === Phase.Done) {
-        // eslint-disable-next-line @typescript-eslint/no-require-imports
-        return require("../../../../../res/img/e2e/verified-deprecated.svg").default;
-    } else {
-        // eslint-disable-next-line @typescript-eslint/no-require-imports
-        return require("../../../../../res/img/e2e/warning-deprecated.svg").default;
-    }
-}
 
 interface IProps {
     onFinished(): void;
 }
-interface IState {
-    icon: string;
-}
-
-export default class SetupEncryptionDialog extends React.Component<IProps, IState> {
-    private store: SetupEncryptionStore;
 
+export default class SetupEncryptionDialog extends React.Component<IProps> {
```

Comment:
if you're basically rewriting a component, consider taking the opportunity to switch to the functional style. Not a blocker here though